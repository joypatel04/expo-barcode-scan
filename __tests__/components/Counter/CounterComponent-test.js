import 'react-native';
import React from 'react';
import Counter from './../../../src/components/Counter';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('Render CounterExample Component Correctly', () => {
    const tree = renderer.create(
        <Counter number={0} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
})
