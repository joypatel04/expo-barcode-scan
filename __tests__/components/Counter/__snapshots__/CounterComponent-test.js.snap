// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render CounterExample Component Correctly 1`] = `
<ScrollView
  contentContainerStyle={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "transparent",
        "flex": 1,
        "justifyContent": "flex-start",
      },
      Object {
        "width": undefined,
      },
    ]
  }
  scrollEnabled={false}
>
  <View
    style={
      Array [
        Object {
          "padding": 10,
        },
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#455A64",
          "fontFamily": "HelveticaNeue",
          "fontSize": 16,
          "fontWeight": "700",
          "letterSpacing": 0,
        }
      }
    >
      Redux Counter Example
    </Text>
  </View>
  <View
    style={
      Array [
        Object {
          "padding": 10,
        },
        Object {
          "top": 10,
        },
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#455A64",
            "fontFamily": "HelveticaNeue",
            "fontSize": 16,
            "fontWeight": "700",
            "letterSpacing": 0,
          },
          Object {
            "fontSize": 120,
          },
        ]
      }
    >
      0
    </Text>
  </View>
  <View
    style={
      Array [
        Object {
          "padding": 10,
        },
        Object {
          "flexDirection": "row",
          "top": 30,
        },
      ]
    }
  >
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "marginLeft": 40,
          "marginRight": 40,
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      />
    </View>
  </View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    isTVSelectable={true}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
        "padding": 10,
        "top": 50,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#455A64",
            "fontFamily": "HelveticaNeue",
            "fontSize": 16,
            "fontWeight": "700",
            "letterSpacing": 0,
          },
          Object {
            "fontSize": 20,
          },
        ]
      }
    >
      RESET
    </Text>
  </View>
</ScrollView>
`;
